website/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "website",
+  "version": "0.0.0",
+  "private": true,
+  "scripts": {
+    "docusaurus": "docusaurus",
+    "start": "docusaurus start",
+    "build": "docusaurus build",
+    "swizzle": "docusaurus swizzle",
+    "deploy": "docusaurus deploy",
+    "serve": "docusaurus serve",
+    "clear": "docusaurus clear"
+  },
```

Comment:
could u explain what those scripts are for? I'm mainly interested in:
- docusaurus
- swizzle
- deploy (if there any config for deployment?)

Reply:
Those scripts are all created by default with a new Docusaurus installation. Here's [the documentation](https://v2.docusaurus.io/docs/cli) on what each does, though it doesn't seem to mention the first `docusaurus` script.

- `swizzle` lets you customize specific theme components
- `deploy` deploys to GH Pages and is documented futher [here](https://v2.docusaurus.io/docs/deployment#deploying-to-github-pages)
  - As the owner, you'll need to set up specific settings so the `deploy` script can run through a GH action or another CI

Reply:
I guess the `docusaurus` just acts as a proxy to its commands and options, so u can do smth like `yarn docusaurus start --flag` or whatever

Thansk for the links!